implementations/rust/ockam/ockam_app/src/enroll/tray_menu.rs

```diff
@@ -1,14 +1,31 @@
 use tauri::{AppHandle, CustomMenuItem, SystemTrayMenu, Wry};
+#[cfg(target_os = "macos")]
+use tauri_runtime::menu::NativeImage;
 
 use crate::app::AppState;
 use crate::enroll::enroll_user::enroll_user;
 
 pub const ENROLL_MENU_HEADER_ID: &str = "enroll-header";
 pub const ENROLL_MENU_ID: &str = "enroll";
+pub const ENROLL_MENU_USER_NAME: &str = "user-name";
 
-pub fn build_enroll_section(app_state: &AppState, tray_menu: SystemTrayMenu) -> SystemTrayMenu {
+pub async fn build_enroll_section(
+    app_state: &AppState,
+    tray_menu: SystemTrayMenu,
+) -> SystemTrayMenu {
     if app_state.is_enrolled() {
-        tray_menu
+        match app_state.get_user_info().await {
+            Some(user_info) => {
+                let item = CustomMenuItem::new(
+                    ENROLL_MENU_USER_NAME,
+                    format!("{} ({})", user_info.name, user_info.nickname),
+                );
+                #[cfg(target_os = "macos")]
+                let item = item.native_image(NativeImage::User);
```

Comment:
Doesn't this unconditionally replace the formatted string, which hadn't been added to the menu yet? Can you caption a NativeImage? If not, wouldn't we probably want both items included on MacOS, in whichever order is appropriate?

Reply:
This just adds an image on the left of the menu title, on MacOs, which is what you can see in the PR description. I think this should also do the right thing on Linux but maybe you can confirm?

Reply:
Gotcha! My primary OS for Ockam work is MacOS, and I can't currently build the Tauri app on my NixOS machine because of the webkit dependencies, which are too new for my configuration. I don't currently have access to another Linux distribution to test with something that is closer to Tauri's documented requirements.